main/guides/js-programming/agoric-js-overview.md

```diff
@@ -198,7 +196,7 @@ sneaky = import
 // tricky comment to obscure function invocation
 (modulename);
 ```
-There are also problems with “import” being near a parenthesis inside a comment. 
+There are also problems when “import” is near a parenthesis inside a comment. 
```

Comment:
Again, it is not clear what direction this problem is. It can only cause a false rejection.